lib/fastlane/plugin/ddg_apple_automation/actions/start_new_release_action.rb

```diff
@@ -21,7 +22,7 @@ def self.run(params)
             params[:github_token], params[:platform], other_action, options
           )
         else
-          release_branch_name, new_version = Helper::DdgAppleAutomationHelper.prepare_release_branch(
+          release_branch_name, new_version, update_embedded_warning = Helper::DdgAppleAutomationHelper.prepare_release_branch(
```

Comment:
Could you rename `update_embedded_warning` to `show_update_embedded_warning` to make it clearer that it’s a boolean value?